PRIVACY.md

```diff
@@ -3,27 +3,48 @@ layout: layouts/privacy.html
 permalink: /privacy/index.html
 ---
 
-# SimpleX Chat Privacy Policy and Conditions of Use
+# SimpleX Chat Operators Privacy Policy and Conditions of Use
 
-SimpleX Chat is the first communication network based on a new protocol stack that builds on the same ideas of complete openness and decentralization as email and web, with the focus on providing security and privacy of communications, and without compromising on usability.
+## Summary
 
-SimpleX Chat communication protocol is the first protocol that has no user profile IDs of any kind, not even random numbers, cryptographic keys or hashes that identify the users. SimpleX Chat apps allow their users to send messages and files via relay server infrastructure. Relay server owners and providers do not have any access to your messages, thanks to double-ratchet end-to-end encryption algorithm (also known as Signal algorithm - do not confuse with Signal protocols or platform) and additional encryption layers, and they also have no access to your profile and contacts - as they do not provide any user accounts.
+[Introduction](#introduction) and [General principles](#general-principles) cover SimpleX Chat network design, the network operators, and the general principles for privacy and security that SimpleX network provides.
+
+[Privacy policy](#privacy-policy) covers:
+- data stored only on your device - [your profiles](#user-profiles), delivered [messages and files](#messages-and-files). This information can be transferred to another device, and you are responsible for its preservation - if you delete the app it will be lost.
```

Comment:
```suggestion
- data stored only on your device - [your profiles](#user-profiles), delivered [messages and files](#messages-and-files). This information can be transferred to another device by user, and you are responsible for its preservation - if you delete the app it will be lost.
```
or "user can transfer"